test/valid-data/type-union/schema.json

```diff
@@ -70,6 +70,27 @@
             ]
           },
           "type": "array"
+        },
+        "var7": {
+          "anyOf": [
+            {
+              "type": "string"
+            },
+            {
+              "const": true,
+              "type": "boolean"
+            }
+          ]
+        },
+        "var8": {
```

Comment:
I'm unsure if we can use `enum` and `type[]` in the same type. Especially the combination of boolean, string, and `true`.

I'd bet that even if allowed, this would break in a lot of community parsers. Could we make this be a anyOf `boolean const true` and `string enum s`?

Reply:
isn’t this the analog case for var4 and var6, apart from using a `number` instead of `boolean`?
regarding whether it is allowed, at least for `type` and `enum` I don’t see a indication that they are exclusive. it feels a bit redundant to specify both, as `enum` implicitly specifies type